### ISSUE TITLE
Dataset and gpu

### ISSUE THREAD

**Issue by Jamalabedi** [SadeghKrmi/pertts-streamlit#10]
Hi!
Regarding the piper model, I wanted to know if I want to train the model with high quality on another voice, how many minutes of sound is needed?
Then, can the modeling be done on the Google colab gpu, or does it require more powerful hardware? 
What GPU did you use for training?
About how many hours/days are needed for complete training?

**Comment by SadeghKrmi** [comment]
Hi,
- between 7 and 8 hours of voice, would be perfect, but 5-6 hours also would be enough.
- No, google colab is not proper for this purpose due to low speed and also periodic interruptions (You can save model after some epochs but still due to lower training speed, there is a lot of challenges to reset the test and wasting some training steps)
- to stop the training process, depends on the training and validation loss, you should monitor and also be careful about the overfit condition as well.
Good luck
